### PR TITLE
feat: update widget version to 2.0.1

### PR DESCRIPTION
**Pull request by dennyscode**
Task: Update Widget to Version 2.0.1

### CODE REVIEW

File: packages/dapp/src/components/Widget/Widget.tsx [hunk 1]
```diff
@@ -140,7 +140,6 @@ export function Widget({ starterVariant }) {
           maxPriceImpact: 0.4,
         },
       },
-      buildSwapUrl: true,
```
Comment (dennyscode) [inline packages/dapp/src/components/Widget/Widget.tsx:143]: @chybisov This "buildSwapUrl" is not used any more and there is no other property which replaces this? Here, I simply deleted it as it is not part of the WidgetConfig types anymore.

File: packages/dapp/src/components/Widgets/Widgets.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { WidgetVariant } from '@lifi/widget';
+import { WidgetSubvariant } from '@lifi/widget';
```
Comment (dennyscode) [inline packages/dapp/src/components/Widgets/Widgets.tsx:1]: Use WidgetSubVariant as Types now

File: packages/dapp/package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "@emotion/styled": "^11.11.0",
     "@lifi/sdk": "^2.0.0-beta.15",
     "@lifi/wallet-management": "2.0.0-beta.8",
-    "@lifi/widget": "^2.0.0-beta.13",
+    "@lifi/widget": "^2.0.1",
```
Comment (dennyscode) [inline packages/dapp/package.json:12]: Version 2.0.1

File: packages/dapp/src/components/Widget/Widget.tsx (hunk 1 above)
Comment (chybisov) [reply packages/dapp/src/components/Widget/Widget.tsx:143]: @dennyscode it was renamed to `buildUrl` 🙂

File: packages/dapp/src/components/Widgets/Widgets.tsx [hunk 4]
```diff
@@ -30,15 +30,15 @@ export function Widgets() {
   const [showWelcome, setShowWelcome] = useState(true);
   const theme = useTheme();
   const [starterVariantUsed, setStarterVariantUsed] = useState(false);
-  const [_starterVariant, setStarterVariant] = useState<WidgetVariant | 'buy'>(
-    'expandable',
-  );
+  const [_starterVariant, setStarterVariant] = useState<
+    WidgetSubvariant | 'buy'
+  >('default');
```
[the remark this answers is not in the document]
Comment (chybisov) [reply packages/dapp/src/components/Widgets/Widgets.tsx:35]: `expandable` is still in place and should be used as part of the `variant` property

**Review by chybisov — APPROVED** [review]

**Pull request merged**